### PR TITLE
chore(web,sdf): remove stable feature flags

### PR DESCRIPTION
**Pull request by paulocsanz**

### CODE REVIEW

File: app/web/src/store/feature_flags.store.ts [hunk 1]
```diff
@@ -3,23 +3,11 @@ import { addStoreHooks } from "@si/vue-lib/pinia";
...
+type FlagStoreKey = "CONTRIBUTE_BUTTON" | "SECRETS";
 
 // The key to this object is the flag name on PostHog, the "storeKey" property is the
 // key used when accessing it from the Pinia feature flag store
 const FLAGS: { [key: string]: { storeKey: FlagStoreKey; default: boolean } } = {
-  one_screen_to_rule_them_all: {
-    storeKey: "SINGLE_MODEL_SCREEN",
-    default: false,
-  },
-  modules_tab: {
```
Comment (zacharyhamm) [inline app/web/src/store/feature_flags.store.ts:19]: We want the modules tab flag to still be in here

**Review by zacharyhamm — COMMENTED** [review]
It looks like you removed the modules_tab flag by accident

File: app/web/src/store/feature_flags.store.ts (hunk 1 above)
Comment (paulocsanz) [reply app/web/src/store/feature_flags.store.ts:19]: Do we? Isn't that enabled for all and stable?

I thought we only wanted to keep the contribute button and the secrets. I can roll it back.

**Review by zacharyhamm — COMMENTED** [review]
@paulocsanz take a spin in the customize panel and open some assets and functions. I'm seeing a lot of UX issues there

**Comment by paulocsanz** [comment]
@zacharyhamm ok now it should be done!

**Review by zacharyhamm — APPROVED** [review]
Nice work! Didn't find any obvious problems. Glad we're cleaning this up quickly

**Comment by paulocsanz** [comment]
bors merge

**Comment by paulocsanz** [comment]
bors merge

**Comment by paulocsanz** [comment]
bors merge

**Pull request merged**